src/schemas/markupmapping.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe. All rights reserved.",
+    "This file is licensed to you under the Apache License, Version 2.0 (the \"License\");",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at http://www.apache.org/licenses/LICENSE-2.0",
+    "",
+    "Unless required by applicable law or agreed to in writing, software distributed under",
+    "the License is distributed on an \"AS IS\" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS",
+    "OF ANY KIND, either express or implied. See the License for the specific language",
+    "governing permissions and limitations under the License."
+  ],
+  "$id": "https://ns.adobe.com/helix/shared/markupmapping",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Markup Mapping",
+  "type": "object",
+  "meta:status": "stabilizing",
+  "description": "",
+  "required": [
+    "match"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "match": {
+      "type": "string",
+      "description": "A CSS selector expression selecting the nodes that should get processed"
+    },
+    "type": {
+      "type": "string",
+      "enum": ["html", "markdown"],
+      "default": "html"
+    },
+    "wrap": {
+      "type": "string",
+      "description": "Add the following HTML tags before the generated HTML"
+    },
+    "class": {
+      "type": "array",
+      "description": "Add the following class names to the `class` attribute of the generated HTML",
+      "items": {
+        "type": "string"
+      }
+    },
+    "attribute": {
```

Comment:
I wonder if this should read `attributes` since it's mostly a map of several attributes